src/contrib/persistence/Akka.Persistence.Sqlite.Tests/Batching/BatchingSqliteEventsByTagSpec.cs

```diff
@@ -6,22 +6,23 @@
 //-----------------------------------------------------------------------
 
 using Akka.Configuration;
+using Akka.Persistence.Query;
 using Akka.Persistence.Query.Sql;
-using Akka.Persistence.Sql.TestKit;
+using Akka.Persistence.TCK.Query;
 using Akka.Util.Internal;
 using Xunit.Abstractions;
 
 namespace Akka.Persistence.Sqlite.Tests.Batching
 {
     public class BatchingSqliteEventsByTagSpec : EventsByTagSpec
     {
-        public static readonly AtomicCounter Counter = new AtomicCounter(200);
+        public static readonly AtomicCounter Counter = new AtomicCounter(0);
```

Comment:
This number was non-zero in order to avoid duplicate connection string used by different tests suites, potentially running in parallel. Maybe we should just linearize sql persistence spec classes via `[Collection]` attribute?